### PR TITLE
Test Plan Updates

### PR DESCRIPTION
**Pull request by brunobowden**
- Min Requirements:
   - iOS 9.0 / iPhone 4S
   - Android 4.4 (API 19) / Samsung Galaxy SII
- Permissions, connectivity, accessibility, installation

## Screenshots

<details>
<summary>Screenshots</summary>
Add any relevant before/after screenshots here
</details>

## How did you test the change?

- [ ] iOS Simulator
- [ ] iOS Device
- [ ] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [ ] other, please describe

## Checklist:

- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

**Comment by brunobowden** [comment]
@Mohit020888 - with the next build, please start doing some testing on these devices

**Comment by brunobowden** [comment]
@Mohit020888 - please review these changes

**Review by Mohit020888 — COMMENTED** [review]
This looks good to me .
The only thing which I would want to add is the high priority Tecno & Infinix devices which are heavily used in Nigeria.

**Comment by Mohit020888** [comment]
> @Mohit020888 - with the next build, please start doing some testing on these devices

Okay Bruno ! 
I am taking the latest build and will try to cover some small screen and low API level devices available on SauceLab

**Comment by brunobowden** [comment]
@Mohit020888 - please do a formal "Approval" on the PR. After this merges, please add an example of the Tecno and Infinix devices. You can replace the `Karbonn Alfa A110`... the thought is to have the lowest powered Android device that still targets Android Kit Kat (4.4).

@theswerd - you'll need to approve also

**Comment by brunobowden** [comment]
@Mohit020888 - our preference would be to set up the testing on Firebase Labs as it's integrated together. I understand if it's easier to start with Sauce Labs but if you think you could do something on Firebase Labs, then I'm happy to figure out giving you access. I think it's only through automation like this that we can get proper testing coverage of enough devices.

File: docs/TESTING.md [hunk 1]
```diff
@@ -2,188 +2,121 @@
...
+| ---------------------------------- | ------------- | ------------------------------------------------------ |
+| **iOS 9.0**                        | Sept 16, 2015 | XCode 12 requires iOS 9.0,<br>Flutter supports iOS 8.0 |
+| **Android 4.4<br>KitKat (API 19)** | Oct 31, 2013  | Flutter 1.22 requirement                               |
 
-#### Android devices
+| Example Device                                                                  |      OS | Release Date |    Screen |      RAM | Note              |
+| ------------------------------------------------------------------------------- | ------: | -----------: | --------: | -------: | ----------------- |
+| [iPhone 4s](https://en.wikipedia.org/wiki/IPhone_4S)                            | iOS 9.0 | Oct 14, 2011 | 960 x 640 |   512 MB |                   |
+| [Samsung Galaxy SII](https://en.wikipedia.org/wiki/Samsung_Galaxy_S_II)         |  API 19 |  May 2, 2011 | 800 x 480 | 1,000 MB |                   |
+| [Karbonn Alfa A110](https://www.gadgetsnow.com/mobile-phones/Karbonn-Alfa-A110) |  API 19 |  Aug 3, 2015 | 480 x 320 | 256 MB\* | India value phone |
```
Comment (Mohit020888) [inline docs/TESTING.md:33]: | Tecno H3](https://parktelonline.com/blog/tecno-h3-price-review-nigeria/ |  API 19 |  Oct 27, 2014 | 320 x 480 | 512 MB\* | Nigeria value phone |

**Review by Mohit020888 — APPROVED** [review]

**Comment by Mohit020888** [comment]
> @Mohit020888 - our preference would be to set up the testing on Firebase Labs as it's integrated together. I understand if it's easier to start with Sauce Labs but if you think you could do something on Firebase Labs, then I'm happy to figure out giving you access. I think it's only through automation like this that we can get proper testing coverage of enough devices.

@brunobowden - I did try running Firebase Robo crawler on our app but it stops working as soon as the control gets out of the app, for ex : Privacy Policy , Terms & Conditions links.
The next solution is to write Flutter Integration Tests which involves writing custom scripts with a robust automation framework.
I don't have the required experience in that involving Flutter as it provides very few locators for automation but an experienced flutter app automation expert would possibly know how to make it work even with limited locators exposed by Flutter.

**Review by theswerd — APPROVED** [review]
LGTM

**Pull request merged**